### DISCUSSION TITLE
.NET MAUI BlazorWebView - how to load a URL at startup

### DISCUSSION THREAD

**Discussion by skyv195** [dotnet/maui#5039] [Q&A]
Hello, I would like to create a mobile application that would load a web url on startup. How do I do this with the BlazorWebView to do this instead of using the default index.html in the project? 

Thank you

**Comment by Xyncgas** [comment]
proceed to add iframe in the component, or use ```NavigationManager.NavigateTo("PageToRedirect");```, or call a javascript function that load url

**Reply by skyv195** [reply]
Thank you so much for the response! Sorry that I am a newbie in this. Can you share a resource on how to add an iframe to the component or how to call `NavigationManager.NavigateTo("PageToRedirect");`? Thank you again!

**Reply by Xyncgas** [reply]
https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.navigationmanager?view=aspnetcore-3.1

**Reply by Xyncgas** [reply]
The MAUI blazor project comes with index.razor, which you can put iframe inside

**Reply by skyv195** [reply]
Thank you again for the speedy response! To streamline the project, can I just add this to the index.razor and remove all the wwwroot html files? I just want the project to autoload a web site URL

```
@page "/" 

<iframe width="560" height="315" src="https://blazor.net" frameborder="0" allowfullscreen></iframe>
```

**Reply by Xyncgas** [reply]
yea, or you could do something like
```
@page "/"
@inject NavigationManager MyNavigationManager
@code{
    protected override OnInitialized()
    {
        MyNavigationManager.NavigateTo("https://www.example.com");
    }
}
```

**Reply by Xyncgas** [reply]
We can add friends on telegram, I am pretty interested in blazor stuffs, we can share stuffs and help out each other

**Reply by skyv195** [reply]
Thank you again for taking the time to help. I will give your approach a try to see if it works..

**Reply by skyv195** [reply]
> We can add friends on telegram, I am pretty interested in blazor stuffs, we can share stuffs and help out each other

Yes, definitely.. My telegram is @andy94211. We can chat more on this subject. Thank you!

**Comment by skyv195** [comment]
Yes, it was the case for me. Not sure if the latest update fixes this.
